plugins/janus_streaming.c

```diff
@@ -590,6 +625,11 @@ int janus_streaming_init(janus_callbacks *callback, const char *config_path) {
 		return -1;
 	}
 
+	struct ifaddrs * ifas = NULL;
+	if(getifaddrs(&ifas) || ifas == NULL) {
+		JANUS_LOG(LOG_ERR, "Unable to acquire list of network devices/interfaces; some configurations may not work as expected...\n");
```

Comment:
Considering this is not returning an error, maybe it should be a `LOG_WARN` instead?